lib/pub_sub/message.rb

```diff
@@ -1,7 +1,7 @@
 module PubSub
   class Message
     def initialize(payload)
-      @payload = JSON.parse(payload)
+      @payload = JSON.parse(JSON.parse(payload)['Message'])
```

Comment:
This to me is a sign that you've tested this with a subscription that doesn't have raw_delivery. This is probably a problem with the way the subscription is set up - we should be setting raw delivery to true when subscribing.

Any queue/topic that is wrapping it's data like that is incorrectly configured.